specification/network/resource-manager/Microsoft.Network/stable/2018-01-01/networkWatcher.json

```diff
@@ -2395,11 +2436,14 @@
       }
     },
     "FlowLogInformation": {
-      "description": "Information on the configuration of flow log.",
+      "description": "Information on the configuration of flow log and traffic analytics (optional).",
       "required": [
         "targetResourceId",
         "properties"
       ],
+      "optional": [
```

Comment:
You don't need to add this section for optional properties. Properties by default are optional.